package.json

```diff
@@ -38,7 +36,7 @@
     "watch-css": "npm run build-css && sass --watch src:src",
     "start-js": "react-scripts start",
     "start": "npm-run-all -p watch-css start-js",
-    "build-js": "react-scripts build",
+    "build-js": "react-scripts --openssl-legacy-provider build",
```

Comment:
Just double-checking here: no upgrade on dependencies was necessary for this NodeJS "bump", right?

All we needed was to add this `openssl` flag to make `react-scripts` compatible with Node 18.

Reply:
Yes, the openssl security issue can be fixed by upgrading the dependencies but it also requires some updates we cannot currently make.
The temporary solution is to enable using the legacy provider until we make the necessary updates.